src/pretix/base/auth.py

```diff
@@ -85,6 +85,16 @@ def authentication_url(self, request):
         """
         return
 
+    def next_url(self, request):
```

Comment:
Can we name this ``get_next_url``? Feels a little more consistent with Django things